easycop.yml

```diff
@@ -0,0 +1,179 @@
+# This file was generated by
+# `rake easycop:init`
+# on 2019-12-16 19:20:52 +0000 using EasyCop version 0.1.
+---
+require:
+  - rubocop-rspec
+AllCops:
```

Comment:
There is a `NewCops` setting that should be set to `false` to avoid warnings on `rubocop` upgrades.

Reply:
Perfect, disabled `NewCops`